Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12.8-alpine
+LABEL maintainer="sberdianskyi@mate.com"
+ENV PYTHONBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
It's a good practice to use `COPY requirements.txt /app/requirements.txt` to ensure the file is copied into the correct directory, especially since the `WORKDIR` is set to `/app`.